browser/src/control/Control.UIManager.js

```diff
@@ -441,21 +441,23 @@ L.Control.UIManager = L.Control.extend({
 	},
 
 	refreshNotebookbar: function() {
-		var selectedTab = $('.ui-tab.notebookbar[aria-selected="true"]').attr('id') || 'Home-tab-label';
-		this.removeNotebookbarUI();
-		this.createNotebookbarControl(this.map.getDocType());
-		if (this._map._permission === 'edit') {
-			$('.main-nav').removeClass('readonly');
+		if (!window.mode.isMobile()) {
```

Comment:
thanks.
I was wondering, if we now need this then it means that when we toggle the theme we are calling this. Wouldn't it be better to just change the `toggleDarkMode` function? After all we already have darkmode related conditions there:

```javascript
	toggleDarkMode: function() {
		// get the initial mode
		var selectedMode = this.getDarkModeState();
		// swap them by invoking the appropriate load function and saving the state
```
[...]
```javascript
		[...]
		else {
			this.refreshNotebookbar();
		}
		this.refreshSidebar();
	},
```

Also this could then be used in the case we would need to (now or in the future) refresh any other mobile-only UI

Reply:
totally agreed. @pedropintosilva how about we seperate this refresh logic from `toggleDarkMode` function so if need refresh thing we can use in other paces as well. 

i have added this condition to `toggleDarkMode` and seprated the refresh logic in other function for more readability and also use cases of that function in future will be more helpful